Example/CollectionViewToolsExample/Views/ImageContentView.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  Copyright © 2020 Rosberry. All rights reserved.
+//
+
+import UIKit
+
+final class ImageContentView: UIView {
+    var removeActionHandler: (() -> Void)?
+
+    let imageView: UIImageView = {
```

Comment:
```suggestion
   private(set) let imageView: UIImageView = {
```
Maybe?

Reply:
`private(set)` modifiers cannot be appllied to `readonly` properties